plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePageSourceProvider.java

```diff
@@ -191,7 +192,9 @@ public static Optional<ConnectorPageSource> createHivePageSource(
         Optional<BucketAdaptation> bucketAdaptation = createBucketAdaptation(bucketConversion, tableBucketNumber, regularAndInterimColumnMappings);
         Optional<BucketValidator> bucketValidator = createBucketValidator(path, bucketValidation, tableBucketNumber, regularAndInterimColumnMappings);
 
-        CoercionContext coercionContext = new CoercionContext(getTimestampPrecision(session));
+        // ORC files treats Double.NaN as null
```

Comment:
nit: is this an ORC spec behaviour or a Apache Hive behaviour, might want to reword this to clarify